modules/@apostrophecms/schema/ui/apos/components/AposArrayEditor.vue

```diff
@@ -329,4 +363,30 @@ export default {
   .apos-modal-array-min-error, .apos-modal-array-max-error {
     color: var(--a-danger);
   }
+
+  .apos-modal-array-items {
+    margin: 10px;
+  }
+
+  .apos-modal-array-items__heading {
+    position: relative;
+    margin: 20px 4px;
+  }
+
+  // Specificity needed due to AposButton rules
+  .apos-modal-array-items__heading .apos-modal-array-items__add {
```

Comment:
Small thing, but since it's an `.apos-button` style that's conflicting, I might do this as `.apos-modal-array-items__add.apos-button` instead to communicate that relationship a bit more.